ts/models/conversation.ts

```diff
@@ -1402,25 +1463,39 @@ export class ConversationModel extends Backbone.Model<ConversationAttributes> {
       this.set('uploadCapability', Boolean(upload));
     }
 
-    if (details.admins && isArray(details.admins)) {
+    const replaceWithOurRealSessionId = async (toReplace: Array<string>) => {
```

Comment:
Maybe we can make this a private method on the class? This way it isn't created everytime set pollinfo is called?

Reply:
moved it to private